eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -488,6 +488,11 @@
           "code": "java\\.annotation\\.removed",
           "old": ".*? com\\.azure\\.monitor\\.query.*",
           "justification": "Removing Jackson annotations from Monitor Query in transition to stream-style."
+        },
+        {
+          "code": "java.field.removed",
+          "old": "field com.azure.resourcemanager.sql.models.StorageCapabilityStorageAccountType.GZRS",
+          "justification": "Remove this because GZRS is deprecated."
```

Comment:
Why we need modify revapi.json?

Reply:
Even if `<revapi.skip>true</revapi.skip>` is added to the pom, the occurrence of ci analyze exception cannot be avoided. The detail info following as : https://dev.azure.com/azure-sdk/public/_build/results?buildId=3782398&view=results